docs/usage.md

```diff
@@ -66,6 +66,10 @@ Please note the following requirements:
 
 Again, by default, the group information is only used to compute co-abundances for the binning step, but not for group-wise co-assembly (see the parameter docs for [`--coassemble_group`](https://nf-co.re/mag/parameters#coassemble_group) and [`--binning_map_mode`](https://nf-co.re/mag/parameters#binning_map_mode) for more information about how this group information can be used).
 
+### Ancient DNA mode
+
+Using the ancient DNA profile `-profile ancient_dna` (see [`profile` section](https://nf-co.re/mag/2.1.0/usage#profile)), you can perform ancient DNA *de novo* assembly validation.
```

Comment:
Maybe woth expanding a bit more what it does here (if you switch from profile to parameter)

```suggestion
Using the ancient DNA profile `-profile ancient_dna` (see [`profile` section](https://nf-co.re/mag/2.1.0/usage#profile)), you can perform ancient DNA *de novo* assembly validation.
```

Reply:
This is outdated, sorry, there is no `aDNA` profile anymore